<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Petanque Turnier</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_0">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabPlayer">
       <attribute name="title">
        <string>Spieler-Liste</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_1">
        <item>
         <widget class="QTreeView" name="tvPlayerList">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <attribute name="headerShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSites">
       <attribute name="title">
        <string>Plätze</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="tlSiteCnt">
            <property name="text">
             <string>Anzahl</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSiteCnt"/>
          </item>
          <item>
           <widget class="QPushButton" name="pbSetSiteCnt">
            <property name="text">
             <string>&amp;Anzahl übernehmen</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="tvSiteList">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRound">
       <attribute name="title">
        <string>Paarungen</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="tlRound">
            <property name="text">
             <string>Runde</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbRound"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="tvMatchList">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPlayerResult">
       <attribute name="title">
        <string>Ergebnisse</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTreeView" name="tvPlayerResultList">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTeamResult">
       <attribute name="title">
        <string>Ergebnisse</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTreeView" name="tvTeamResultList">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Datei</string>
    </property>
    <addaction name="actionBackup"/>
    <addaction name="actionSettings"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPlayer">
    <property name="title">
     <string>&amp;Spieler</string>
    </property>
    <addaction name="actionPlayerAdd"/>
    <addaction name="separator"/>
    <addaction name="actionPlayerLoad"/>
    <addaction name="actionPlayerSave"/>
   </widget>
   <widget class="QMenu" name="menuRound">
    <property name="title">
     <string>&amp;Runde</string>
    </property>
    <addaction name="actionRoundCreate"/>
    <addaction name="actionRoundFinish"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Hilfe</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;Ansicht</string>
    </property>
    <addaction name="actionCreateWindow"/>
    <addaction name="actionDeleteAllWindows"/>
    <addaction name="separator"/>
    <addaction name="actionLockScreen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlayer"/>
   <addaction name="menuRound"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPlayerLoad">
   <property name="text">
    <string>Spieler-Liste &amp;laden...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPlayerSave">
   <property name="text">
    <string>Spieler-Liste &amp;speichern...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Ende</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPlayerPrint">
   <property name="text">
    <string>Liste drucken</string>
   </property>
  </action>
  <action name="actionRoundCreate">
   <property name="text">
    <string>Runde &amp;erzeugen</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRoundFinish">
   <property name="text">
    <string>Runde &amp;beenden</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionBackup">
   <property name="text">
    <string>&amp;Sicherung erstellen...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPlayerAdd">
   <property name="text">
    <string>Spieler &amp;hinzufügen...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRoundPrint">
   <property name="text">
    <string>Runde drucken</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Einstellungen...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Über...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCreateWindow">
   <property name="text">
    <string>Neues Fenster erzeugen...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDeleteAllWindows">
   <property name="text">
    <string>Alle erzeugten Fenster schließen</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMarkAllPlayers">
   <property name="text">
    <string>Alle Markierungen setzen</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionUnmarkAllPlayers">
   <property name="text">
    <string>Alle Markierungen entfernen</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSimulatePlayers">
   <property name="text">
    <string>Simulierte Spieler erzeugen</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMarkAllSites">
   <property name="text">
    <string>Alle Markierungen setzen</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionUnmarkAllSites">
   <property name="text">
    <string>Alle Markierungen entfernen</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSimulateResults">
   <property name="text">
    <string>Simulierte Ergebnisse erzeugen</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionLockScreen">
   <property name="text">
    <string>Bildschirm sperren</string>
   </property>
  </action>
  <action name="actionExportWindow">
   <property name="text">
    <string>Daten exportieren...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
